### ISSUE TITLE
Give inventory a flair slot

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoSuggestions#988]
While it's nice to show off your meteorite shard, it taking an inventory slot hampers you a little through the game. It would be nice if the inventory had a flair slot where the meteorite shard could go. Not to make it exclusive to that, maybe we could add a dew other flairs - maybe a ghost for holloween, or the planet, etc.

**Comment by Elliander** [comment]
I agree. With all the perks from being a Kickstarter backer, it leaves initial storage space more than a little cluttered then goes straight into a chest if not worn.
